spec/lib/myfinance/resources/payable_account_spec.rb

```diff
@@ -110,4 +110,19 @@
       end
     end
   end
+
+  describe "#undo_payment", vcr: true do
+    subject { client.payable_accounts.undo_payment(1215631, entity_id) }
+
+    it "undoes receivement of the receivable account" do
+      expect(subject.id).to eq(1215631)
+      expect(subject.status).to eq(1)
+      expect(subject.status_name).to eq('unpaid')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.